src/game/server/instagib/chat_commands.cpp

```diff
@@ -483,6 +483,56 @@ void CGameContext::ConRankFlagCaptures(IConsole::IResult *pResult, void *pUserDa
 	pSelf->m_pController->m_pSqlStats->ShowRank(pResult->m_ClientId, pName, "Flag captures", "flag_captures", pSelf->m_pController->StatsTable(), "DESC");
 }
 
+void CGameContext::ConTopSpikeColors(IConsole::IResult *pResult, void *pUserData)
+{
+	const auto *pSelf = static_cast<CGameContext *>(pUserData);
+	if(!CheckClientId(pResult->m_ClientId))
+		return;
+
+	if(!pSelf->m_pController)
+		return;
+
+	if(pSelf->m_pController->IsDDRaceGameType())
```

Comment:
This should check for fng not for ddrace gametype. There are no spikes in zCatch or ctf for example.